MEAnalysis/python/MEAnalysis_cfg_heppy.py

```diff
@@ -300,6 +300,13 @@ class Conf:
             "dl_jge4_t2": 20,
             "dl_jge4_t3": 2.3,
             "dl_jge4_tge4": -20,
+
+            ##[CHECK-ME]
+            "fh_j8_t4": -20,
+            "fh_j7_t4": -20,
+            "fh_j7_t3": -20,
+            "fh_jge6_t4": -20,
+            "fh_jge6_t3": -20,
         },
```

Comment:
I would add fh_j8_t3 and fh_j9_t4 for now. We will need to define the BLR cuts later.